tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"lib/helper",
+	"globalize/date"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
`external/date.js` calls `attributes.parse( date );`, yet this doesn't export a `parse` function, seems fishy. Missing code coverage?